docs/command-line/promtool.md

```diff
@@ -592,6 +592,33 @@ Dump samples from a TSDB.
 
 
 
+##### `promtool tsdb dump-series`
+
+Dump duplicate series (remove samples content) from a TSDB.
+
+
+
+###### Flags
+
+| Flag | Description | Default |
+| --- | --- | --- |
+| <code class="text-nowrap">--sandbox-dir-root</code> | Root directory where a sandbox directory would be created in case WAL replay generates chunks. The sandbox directory is cleaned up at the end. | `data/` |
+| <code class="text-nowrap">--min-time</code> | Minimum timestamp to dump. | `-9223372036854775808` |
+| <code class="text-nowrap">--max-time</code> | Maximum timestamp to dump. | `9223372036854775807` |
```

Comment:
Can you explain a little better how these impact the output?
Is it a feature you really need, to control this?

Reply:
Documentation information automatically generated using the `make cli-documentation` command :)